schedule/src/main/java/com/droidcon/schedule/ui/viewmodel/ScheduleFragmentViewModel.kt

```diff
@@ -12,11 +13,11 @@ class ScheduleFragmentViewModel @Inject constructor(
     private val getSessionsPerDay: GetSessionsPerDay
 ) : ViewModel() {
 
-    val sessions = liveData {
+    val sessions = liveData(Dispatchers.IO) {
```

Comment:
I think it would be convenient to pass the dispatcher through constructor, but we can do it later